mx.mx/suite.py

```diff
@@ -110,6 +110,14 @@
       "sourceUrls" : ["https://lafo.ssw.uni-linz.ac.at/pub/jmh/jmh-runner-1.12-sources.jar"],
       "licence" : "GPLv2-CPE",
     },
+
+    "JACKPOT" : {
+      "path" : "lib/jackpot-3.0-8.1-SNAPSHOT.jar",
```

Comment:
You should omit the `"path"` attribute so that mx can place the download in its cache (i.e., `~/.mx/cache`)